### DISCUSSION TITLE
If I log in to Kibana through Security Onion and navigate to the Security page, it shows "Page Not Found." why? Please suggest

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#13969] [Unsupported Versions]
Hello ,
My Security Onion version is 2.4.100, deployed in the cloud. I performed a network installation on a standalone instance with 4 vCPUs, 16 GB RAM, and a 256 GB root volume. The so-status command shows that all services are running, and sudo salt-call state.highstate executed without any errors.
If I log in to Kibana through Security Onion and navigate to the Security page, it shows "Page Not Found." Why is this happening?
![image](https://github.com/user-attachments/assets/92669bfa-c953-40b3-ae94-9a9f8e63df9c)
Only the Security page displays this error, while the Observability and Analytics pages are working fine.

**Comment by InfosecGoon** [comment]
How exactly are you getting to this page?

**Reply by sandraimmaculate** [reply]
Thanks for your reply
I will be accessing it via Security Onion > Kibana > Home > Security.
Is this page is only for licensed version

**Reply by InfosecGoon** [reply]
That's part of Kibana that I think requires a separate license from Elastic -- it's not enabled in Security Onion. My test box is showing the same thing.
